discord-groups/style.css

```diff
@@ -28,16 +30,23 @@
   justify-content: start;
   gap: 1rem;
   flex-direction: row;
-  box-shadow: 4px 4px 5px var(--color-groups-tab-box-shadow);
+  /* box-shadow: 4px 4px 5px var(--color-groups-tab-box-shadow); */
   background-color: var(--color-groups-tab-background);
 }
 .groups-tab div {
   font-size: larger;
+  font-weight: bolder;
+  color: #fff;
   cursor: pointer;
-  padding: 1rem;
+  margin: 1.5rem;
+  padding-bottom: 0.5rem;
+}
+.groups-tab div.active-tab {
+  color: var(--color-active-tabs-color);
+  border-bottom: 4px solid var(--color-active-tabs-color);
 }
-.active-tab {
-  background-color: var(--color-active-tabs-background);
+.groups-tab div:hover {
+  color: var(--color-active-tabs-color);
```

Comment:
please use classNames, instead of div give it a className and use it

Reply:
used classnames for elements